unlock-app/src/components/interface/checkout/alpha/Checkout/Metadata.tsx

```diff
@@ -182,7 +182,7 @@ export function Metadata({ checkoutService, injectedProvider }: Props) {
                 >
                   {hideRecipient ? (
                     <div className="space-y-1">
-                      <div className="ml-1 text-sm"> Recipient #1 </div>
```

Comment:
Should it be removed from the subsequent ones as well?

Because if yes, you need to make a change below as well.

Reply:
Fixed it!